design/images/component-api-and-auth.puml

```diff
@@ -1,42 +1,40 @@
-@startuml
-!includeurl https://raw.githubusercontent.com/RicardoNiepel/C4-PlantUML/master/C4_Component.puml
+@startuml component-api-and-auth
+' C4 Component diagram for the API and Authentication modules
+!include https://raw.githubusercontent.com/plantuml-stdlib/C4-PlantUML/master/C4_Component.puml
 
-' C4 Component diagram for The Seedcase product - API
-
-Person(direct_access, "Direct connection")
-Person(admin, "Admin user")
 Container(frontend, "Front end")
-Container(backend, "Back end")
-Container(uia, "User ID Authentication")
+Container(bee, "Back end environment")
```

Comment:
I think Backend is one word

Reply:
If that's the case, it should be changed in the text as well